docs/reference/database-migration.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Database Migration
+description: A guide to migrating lakeFS database.
+parent: Reference
+nav_order: 51
+has_children: false
+---
+
+# lakeFS Database Migrate
+{: .no_toc }
+
+**Note:** Feature in development
+{: .note }
+
+lakeFS database migration allows users to easily transition from one database implementation to another.
```

Comment:
Maybe:
The lakeFS database migration tool makes it easy for users to switch from one database implementation to another.

Reply:
@nopcoder See updated upgrade comment